modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/fragment/renderer/FDSViewFragmentRenderer.java

```diff
@@ -437,16 +437,55 @@ private JSONArray _getFiltersJSONArray(
 				String type = MapUtil.getString(properties, "type");
 
 				if (Objects.equals(type, "date")) {
+					boolean hasPreloadedData;
+
+					String from = MapUtil.getString(properties, "from");
+					String to = MapUtil.getString(properties, "to");
```

Comment:
Do we need to convert this to string? For a `null` check, `Object from = ...` should is enough, and we can reuse the variable for `_getDateJSONObject(from)`